app/controllers/concerns/insights/api/common/tagging_methods.rb

```diff
@@ -0,0 +1,38 @@
+module Insights
+  module API
+    module Common
+      module TaggingMethods
+        def tag
+          primary_instance = primary_collection_model.find(request_path_parts["primary_collection_id"])
+
+          applied_tags = parsed_body.collect do |i|
+            begin
+              tag = Tag.find_or_create_by!(Tag.parse(i["tag"]))
+              primary_instance.tags << tag
+              i
+            rescue ActiveRecord::RecordNotUnique
+            end
+          end.compact
+
+          # HTTP Not Modified
+          return head(304, :location => "#{instance_link(primary_instance)}/tags") if applied_tags.empty?
```

Comment:
move from status names to number was because of rubocop ?

Reply:
Yes, https://github.com/ManageIQ/guides/pull/363